modules/rostests/apitests/atl/CSimpleArray.cpp

```diff
@@ -14,31 +14,31 @@
 #include <atlbase.h>
 #include <atlsimpcoll.h>
 
-struct CCreature
+struct CSimpleCreature
```

Comment:
In theory you don't have to rename the class, can just drop it in an anonymous namespace.
Either way works.

Reply:
I'm a simple creature. I rename things. 🙂

Reply:
Is there any kind of name-collision happening? Or why does it have to be renamed? (I couldn't quickly understand that from the PRs description)

Reply:
ODR